EIPS/eip-2774.md

```diff
@@ -0,0 +1,129 @@
+---
+eip: 2774
+title: A price-sensitive challenge process 
+author: Patrick McCorry <@stonecoldpat>
+discussions-to: https://github.com/ethereum/EIPs/issues/2775
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-07-07
```

Comment:
I believe you need specify that you are depending on EIP-1559 -- in that case, you'd need to add `requires: 1559` to your preamble.